docs/sphinx/using/backends/hardware.rst

```diff
@@ -112,6 +112,128 @@ To see a complete example for using Amazon Braket backends, take a look at our :
+to specify which machine to submit quantum kernels to:
+
+.. code:: bash
+
+    nvq++ --target infleqtion --infleqtion-machine cq_sqale_qpu src.cpp ...
+
+where ``cq_sqale_qpu`` is an example of a physical QPU.
+
+To run an ideal dry-run execution on the QPU, additionally pass ``dry-run`` with the ``--infleqtion-method`` 
+flag to the ``nvq++`` compiler:
+
+.. code:: bash
+
+    nvq++ --target infleqtion --infleqtion-machine cq_sqale_qpu --infleqtion-method dry-run src.cpp ...
+
+To noisily simulate the QPU instead, pass ``noise-sim`` to the ``--infleqtion-method`` flag like so:
+
+.. code:: bash
+
+    nvq++ --target infleqtion --infleqtion-machine cq_sqale_qpu --infleqtion-method noise-sim src.cpp ...
+
+Alternatively, to emulate the Infleqtion machine locally, without submitting through the cloud,
+you can also pass the ``--emulate`` flag to ``nvq++``. This will emit any target
+specific compiler diagnostics, before running a noise free emulation.
+
+.. code:: bash
+
+    nvq++ --emulate --target infleqtion src.cpp
+
+To see a complete example for using Infleqtion's backends, take a look at our :doc:`C++ examples <../examples/examples>`.
+
+Submission from Python
+`````````````````````````
+
+The target to which quantum kernels are submitted
+can be controlled with the ``cudaq::set_target()`` function.
+
+.. code:: python
+
+    cudaq.set_target("infleqtion")
+
+By default, quantum kernel code will be submitted to Infleqtion's Sqale
+simulator.
+
+To specify which Infleqtion QPU to use, set the :code:`machine` parameter.
+
+.. code:: python
+
+    cudaq.set_target("infleqtion", machine="cq_sqale_qpu")
+
+where ``cq_sqale_qpu`` is an example of a physical QPU.
+
+To run an ideal dry-run execution of the QPU, additionally set the ``method`` flag to ``"dry-run"``.
+
+.. code:: python
+
+    cudaq.set_target("infleqtion", machine="cq_sqale_qpu", method="dry-run")
+
+To noisily simulate the QPU instead, set the ``method`` flag to ``"noise-sim"``.
+
+.. code:: python
+
+    cudaq.set_target("infleqtion", machine="cq_sqale_qpu", method="noise-sim")
+
+Alternatively, to emulate the Infleqtion machine locally, without submitting through the cloud,
+you can also set the ``emulate`` flag to ``True``. This will emit any target
+specific compiler diagnostics, before running a noise free emulation.
+
+.. code:: python
+
+    cudaq.set_target("infleqtion", emulate=True)
+
+The number of shots for a kernel execution can be set through
+the ``shots_count`` argument to ``cudaq.sample`` or ``cudaq.observe``. By default,
+the ``shots_count`` is set to 1000.
+
+.. code:: python
+
+    cudaq.sample(kernel, shots_count=100)
+
+To see a complete example for using Infleqtion's backends, take a look at our :doc:`Python examples <../examples/examples>`.
+Moreover, for an end-to-end application workflow example executed on the Infleqtion QPU, take a look at the 
+:doc:`AIM ground state solver <../applications/python/logical_aim_sqale>` notebook.
```

Comment:
This link doesn't work as is. I suggest to instead link to the applications page, and list teh new AIM ground state solver there.
```suggestion
:doc:`AIM ground state solver <../applications>`.
```

Reply:
I took the liberty of pushing a commit that does that.